patching/src/main/java/org/jboss/as/patching/tool/PatchingHistory.java

```diff
@@ -426,7 +426,7 @@ public Patch getMetadata() {
                                     try {
                                         patch = ((PatchBuilder)PatchXml.parse(patchXml)).build();
                                     } catch (Exception e) {
-                                        e.printStackTrace(); // TODO
+                                        PatchLogger.ROOT_LOGGER.error(e.getLocalizedMessage(), e);
```

Comment:
Please leave the // TODO bit unless @aloubyansky has decided he doesn't want it any more.

Reply:
You can remove the TODO, thanks!